velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -195,8 +195,15 @@ class StringTest : public SparkFunctionBaseTest {
 TEST_F(StringTest, Ascii) {
   EXPECT_EQ(ascii(std::string("\0", 1)), 0);
   EXPECT_EQ(ascii(" "), 32);
-  EXPECT_EQ(ascii("😋"), -16);
+  EXPECT_EQ(ascii("😋"), 128523);
   EXPECT_EQ(ascii(""), 0);
+  EXPECT_EQ(ascii("¥"), 165);
+  EXPECT_EQ(ascii("®"), 174);
+  EXPECT_EQ(ascii("©"), 169);
+  EXPECT_EQ(ascii("VELOX"), 86);
```

Comment:
Might be helpful to add a couple more test cases that start with V to illustrate that results depend only on first character.

Reply:
Just added 3 more cases. Thanks!

Reply:
Thanks.